code/datums/mind.dm

```diff
@@ -74,13 +74,17 @@
 
 	SSnano.user_transferred(current, new_character) // transfer active NanoUI instances to new user
 
+	if(cm_skills)
+		cm_skills.omnighost = FALSE
+		current.verbs -= /mob/proc/check_skills
```

Comment:
Skills datum are tied to the mind, not mobs.
This is done with the intention of potentially allowing any "skilled" mob to review their skills, while still preventing the ones without. Instead of just tying this up to humans only (ergo human ghosts should be able to use the verb too now).